src/q7/matrix-INMA2380/exercises/matrix-INMA2380-exercises.tex

```diff
@@ -16,7 +19,7 @@
 \usepackage{url}
 
```

Comment:
Add
```
\vfuzz=318pt
```
so that exercise 1.8 stops giving `overfull \vbox` errors. The value 318 comes from the largest value given on the output, rounded towards +oo.

The problem is that the matrices are too large to fit within the previous page (where the statemen is located, page 8), so LaTeX puts the matrices on the next page (page 9), along with the single line of text above (in the same paragraph), and along with the `solution`'s upper curve. But this leaves a large amount of blank space on the bottom of page 8, exceeding LaTeX's acceptable level, and thus it cries.

In a sense, this is a "shut up" fix, but this is mostly the only reasonable fix for this warning, outside of rewording page 8 to fill in the blank space (either adding words or removing words).

Reply:
(However, the following warnings are present then:
```
Package mdframed Info: Box was splittet wrong
 starting loop to iterate the splitting point
(mdframed)              on input line 260.


Package mdframed Warning: correct box splittet fails
 It seems you are using a non splittable contents
(mdframed)                 on input line 260.

Package mdframed Info: You first box width is to small
 mdframed fixed it
(mdframed)              on input line 260.
[7]
```
Mostly because `mdframed` still has difficulty of letting this go. So we traded an `overfull \vbox` for a `mdframed` warning...)

Reply:
Reworded, no longer a problem.